bilingual-module/README

```diff
@@ -31,44 +31,51 @@ Compiling
 
 Given the requirements being installed, you should be able to just run:
 
-    $ ./configure
-    $ make
-    # make install
+```console
+$ ./configure
+$ make
+# make install
+```
 
-You can use `./autogen.sh` instead of `./configure` you're compiling from
-source. If you installed any prerequisite language packages using a `--prefix`
-to `./configure`, make sure to give the same `--prefix` to `./configure` here.
+You can use `./autogen.sh` instead of `./configure` in case you're compiling from
+source.
```

Comment:
Do you mind re-wrapping this to be 80-columns?

Reply:
Sure